src/templates/about.pug

```diff
@@ -3,25 +3,25 @@ block content
   .contentContainer
     .aboutHeader The Handshake Alliance
     .imgParagraph
-      img(src="/img/logo.svg").aboutImg
+      img(src="/img/HAlogo.svg").aboutImg
       p.aboutParagraph We are a coalition of companies and developers building open source tools and infrastructure for the Handshake network. Our mission is to expedite the adoption of handshake and see its power in the hands of the ordinary consumer.
     .aboutHeader Our Block Explorer
     .imgParagraph.inverse
       img(src="/img/3d.svg").aboutImg
       p.aboutParagraph
-        | A truly open source solution with simple and intuitive navigation, HNScan provides developers a free device to delve into the block chain and examine blocks, transactions and namespaces. We are always looking for 
-        a(href="https://github.com/HandshakeAlliance/HNScan" target="_blank" rel="noopener noreferrer") help 
+        | A truly open source solution with simple and intuitive navigation, HNScan provides developers a free device to delve into the block chain and examine blocks, transactions and namespaces. We are always looking for
+        a(href="https://github.com/HandshakeAlliance/HNScan" target="_blank" rel="noopener noreferrer") help
         | to improve its functionality.
     .aboutHeader  Innovation's Finest
     .imgParagraph
       img(src="/img/smartphone-call.svg").aboutImg
       p.aboutParagraph
-        | If you wish to count yourself among some of crypto's brightest minds, join the 
```

Comment:
I'm not certain on this, but I know Pug's whitespace can act a little weird occasionally.  It _might_ still need the spaces (or be refactored to have the spaces we need). Can you check if the about page looks right?